packages/create-plugin/src/utils/utils.packageManager.ts

```diff
@@ -0,0 +1,105 @@
+import { basename } from 'path';
+import { spawnSync } from 'child_process';
+import { sync as findUpSync } from 'find-up';
+import { getPackageJson } from './utils.npm';
+
+const NPM_LOCKFILE = 'package-lock.json';
+const PNPM_LOCKFILE = 'pnpm-lock.yaml';
+const YARN_LOCKFILE = 'yarn.lock';
+const SUPPORTED_PACKAGE_MANAGERS = ['npm', 'yarn', 'pnpm'];
+const LEGACY_PACKAGE_MANAGER = { packageManagerName: 'yarn', packageManagerVersion: '1.22.19' };
```

Comment:
Nit: `FALLBACK_PACKAGE_MANAGER`. or `DEFAULT_PACKAGE_MANAGER` we are not really deprecating yarn 1. just making it the default among other options.

Reply:
Replaced with `DEFAULT_PACKAGE_MANAGER`. 👍